static/app/views/performance/database/getIntervalForMetricFunction.tsx

```diff
@@ -19,8 +21,21 @@ export function getIntervalForMetricFunction(
   metricFunction: Aggregate | SpanFunctions | string,
   datetimeObj: DateTimeObject
 ) {
+  const sentryTransaction = Sentry.getCurrentHub().getScope()?.getTransaction();
+
+  const sentrySpan = sentryTransaction?.startChild({
+    op: 'function',
+    description: 'getIntervalForMetricFunction',
+    data: {
+      ...datetimeObj,
+    },
+  });
+
   const ladder = GRANULARITIES[metricFunction] ?? COUNTER_GRANULARITIES;
-  return ladder.getInterval(getDiffInMinutes(datetimeObj));
+  const interval = ladder.getInterval(getDiffInMinutes(datetimeObj));
```

Comment:
this can just be 
```suggestion
const interval = Sentry.startSpan(
  {op: 'function', name: 'getIntervalForMetricFunction', data: {...datetimeObj}},
  () => ladder.getInterval(getDiffInMinutes(datetimeObj))
);
```

https://docs.sentry.io/platforms/javascript/performance/instrumentation/custom-instrumentation/#start-span